pages/common/zipalign.md

```diff
@@ -0,0 +1,12 @@
+# zipalign
+
+> Zip archive alignment tool.
+> More information: <https://developer.android.com/studio/command-line/zipalign>.
+
+- Align the data of a ZIP file on {{number}} bytes:
+
+`zipalign {{number}} {{path/to/input.zip}} {{path/to/output.zip}}`
+
+- Check that a ZIP file is correctly aligned on {{number}} bytes:
+
+`zipalign -v -c {{number}} {{path/to/input.zip}}`
```

Comment:
`-v` is for verbose right ? We should also mention that in the description.

Reply:
Indeed. (Because zipalign is quiet by default and without it, it would just return an exit code.)

Reply:
You have not mentioned the verbosity part in the description.

Reply:
Fair enough, I added your suggestion.